lib/puppet-languageserver/handlers.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+%w[
+  handlers/pdk_handler.rb
```

Comment:
the intention here is to have more than just a pdk handler. add classes that handle invoking the providers and other classes that message_router has, to reduce the amount of code there